appveyor.yml

```diff
@@ -62,32 +61,34 @@ assembly_info:
   assembly_file_version: "{version}"
   assembly_informational_version: "{version}"
 
-#---------------------------------#
-#       build configuration       #
-#---------------------------------#
-
 configuration: Release
 
-# scripts to run before build
 before_build:
   - cd C:\projects\nupic-core\build\scripts
   - cmake C:\projects\nupic-core\src -G "Visual Studio 14 2015 Win64" -DCMAKE_INSTALL_PREFIX=..\Release -DLIB_STATIC_APR1_LOC=..\..\external\windows64\lib\apr-1.lib -DLIB_STATIC_APRUTIL1_LOC=..\..\external\windows64\lib\aprutil-1.lib -DLIB_STATIC_YAML_CPP_LOC=..\..\external\windows64\lib\yaml-cpp.lib -DLIB_STATIC_YAML_LOC=..\..\external\windows64\lib\yaml.lib -DLIB_STATIC_Z_LOC=..\..\external\windows64\lib\z.lib  -DLIB_STATIC_CAPNP_LOC=..\..\external\windows64\lib\capnp.lib  -DLIB_STATIC_KJ_LOC=..\..\external\windows64\lib\kj.lib
 
-# To run your custom scripts instead of automatic MSBuild
 build_script:
-  - msbuild "C:\projects\nupic-core\build\scripts\ALL_BUILD.vcxproj" /p:Configuration=Release /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
+  - msbuild "C:\projects\nupic-core\build\scripts\gtest.vcxproj" /p:Configuration=Release /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
```

Comment:
Is not supposed that `ALL_BUILD.vcxproj` build all these listed projects? Why it is being removed?